ricecooker/managers/tree.py

```diff
@@ -161,10 +125,11 @@ def reattempt_failed(self, failed):
             config.LOGGER.info("\tReattempting {0}".format(str(node[1])))
             for f in node[1].files:
                 # Attempt to upload file
-                with  open(config.get_storage_path(f['filename']), 'rb') as file_obj:
-                    response = config.SESSION.post(config.file_upload_url(),  files={'file': file_obj})
-                    response.raise_for_status()
-                    self.uploaded_files += [f['filename']]
+                if f.filename:
```

Comment:
I think you already do validation and/or filtering on this above, so maybe OK to skip this check (or do it as an `assert` here)?